### ISSUE TITLE
Reducing substance high in stool

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11199]
**Preferred term label:**
Reducing substance high in stool

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1535977

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0031979

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/610370

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001295
- label: Elevated stool reducing substances
- parents: [Abnormal stool composition (HP:0031685)](https://hpo.jax.org/app/browse/term/HP:0031685)
- pmid: 
- def: Elevated level of reducing substances in the feces. If carbohydrate malabsorption is suspected, a reducing substance should be investigated in the stool using the Benedict or Fehling test. The presence of reducing substances may indicate an abnormality in carbohydrate (e.g., sucrose, lactase) metabolism. The unabsorbed sugars in stool are measured as reducing substances.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11199
- synonyms: Elevated fecal reducing substances(NONE)|Increased reducing substances in stool(NONE)
